### ISSUE TITLE
About image normalization

### ISSUE THREAD

**Issue by YudeWang** [leftthomas/SuperResolution#30]
Thanks for your work! But I have a tiny question for image normalization.
https://github.com/leftthomas/ESPCN/blob/b1dd698406b0a6dbdbb9c2618bbb06dc903e9b06/test_image.py#L45
The output of model is in range [0,1], but I didn't find the normalization for input and target during training, which are loaded from file in range [0, 255]. Then there will be some trouble for the loss between model output and the target. Can you help me to figure out where is the normalization for the input and target to change the range from [0, 255] into [0,1]?

**Comment by leftthomas** [comment]
@YudeWang 
https://github.com/leftthomas/ESPCN/blob/b1dd698406b0a6dbdbb9c2618bbb06dc903e9b06/test_image.py#L38
You need learn PyTorch better
